core/src/main/scala/com/nvidia/spark/rapids/tool/qualification/PluginTypeChecker.scala

```diff
@@ -83,7 +85,13 @@ class PluginTypeChecker extends Logging {
   def getSupportedExprs: Map[String, String] = supportedExprs
 
   private def readOperatorsScore: Map[String, Double] = {
-    val source = Source.fromResource(OPERATORS_SCORE_FILE)
+    var file: String = null
+    platform match {
+      case "dataproc" => file = OPERATORS_SCORE_FILE_DATAPROC
+      case "emr" => file = OPERATORS_SCORE_FILE_EMR
+      case _ => file = OPERATORS_SCORE_FILE_ONPREM
+    }
+    val source = Source.fromResource(file)
```

Comment:
it might be nice to log a line saying which platform was actually used.  We could potentially add to csv but I would say utnil we find a need a log line would be sufficient

Reply:
added logInfo statement in latest commit